### DISCUSSION TITLE
About torch.view

### DISCUSSION THREAD

**Discussion by Aniruddha120** [mrdbourke/pytorch-deep-learning#392] [Q&A]
Hello Guys, Hope this message find you well!

I was watching the tutorial of Pytorch of Daniel and at 3:08:39 while working with torch.view, I have found that the z tensor have two square brackets,(Pls look at the ss) but the x does not have two. Can you pls tell me what does it mean??

Thanks in advance!!

![image](https://user-images.githubusercontent.com/34673269/230728951-00d76a4a-b35f-40d4-8c79-6fe0fe587539.png)

**Comment by alaminifty** [comment]
I'm also looking for the same thing!

**Comment by Aniruddha120** [comment]
Solved!
